src/components/SigningModals/TransactionStepper/useStepperLayout.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useRef, useState } from 'react'
+import useMeasure from 'react-use-measure'
+
+function useStepperLayout() {
+  const [outerBoundsRef, outerBounds] = useMeasure()
+  const innerBoundsRef = useRef()
+  const [innerBounds, setInnerBounds] = useState(null)
+  const [measuring, setMeasuring] = useState(true)
```

Comment:
Exposing this flag allows us to control the presentation of the stepper at the frame just before the measure and switch has happened and avoid an awkward flash while we measure and decide to collapse the stepper.

Reply:
Worth adding a comment!